subprojects/base-services/src/test/groovy/org/gradle/internal/service/AccessModifierDefaultServiceRegistryTest.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.service
+
+import org.gradle.internal.service.internal.ModifierStubs
+import spock.lang.Specification
+
+class AccessModifierDefaultServiceRegistryTest extends Specification {
```

Comment:
Is there existing coverage for a service with a default constructor?

Reply:
Yes

Reply:
I added another one for sanity's sake